src/app/app.routing.ts

```diff
@@ -2,14 +2,21 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { BrowserModule } from '@angular/platform-browser';
 import { RouterModule, Routes } from '@angular/router';
-import { AuthLayoutComponent } from './layouts/auth-layout/auth-layout.component';
-import { CommonLayoutComponent } from './layouts/common-layout/common-layout.component';
-import { PasswordResetComponent } from './pages/password-reset/password-reset.component';
-import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
+import { AuthenticationComponent } from './modules/authentication/authentication.component';
+import { FeaturesComponent } from './modules/features/features.component';
+import { PasswordResetComponent } from './modules/authentication/components/password-reset/password-reset.component';
+import { EnterResetPasswordComponent } from './modules/authentication/components/enter-reset-password/enter-reset-password.component';
 import { NoAuthGuard } from './guards/no-auth.guard';
 import {
   InternationalPaymentStatusComponent
-} from 'src/app/pages/payment/international-payment-status/international-payment-status.component';
+} from 'src/app/modules/features/components/payment/international-payment-status/international-payment-status.component';
+import { PropertyDetailsComponent } from './components/property-details/property-details.component';
+import { NoPropertiesComponent } from './components/properties/no-properties/no-properties.component';
+import { TermsPageComponent } from './components/terms/terms.component';
+import { PropertiesComponent } from './components/properties/properties.component';
+import { RegistersuccessComponent } from './modules/authentication/components/registration/registersuccess/registersuccess.component';
+import { HomeComponent } from './components/home/home.component';
+import { AuthGuard } from './guards/auth.guard';
 
 const routes: Routes = [
   {
```

Comment:
Is the empty bracket necessary, kindly enlight

Reply:
@Manorlds-Eaglespark , I'm not seeing empty brackets, it's just that it hasn't been modified
![image](https://user-images.githubusercontent.com/12128153/64607695-b207ec00-d3d1-11e9-9bca-6b16f642c4a8.png)

Reply:
there is actually empty bracket here... maybe i will sync with you on this and you tell me what is happening